tests/dir.sdkbuild.props

```diff
@@ -10,6 +10,7 @@
     <RuntimeFrameworkVersion>$(MicrosoftNETCoreAppPackageVersion)</RuntimeFrameworkVersion>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <Platform>AnyCPU</Platform>
```

Comment:
Is this property still used in SDK projects? I always thought of this as a very VS specific property.

Reply:
Yes, it's still used. By default in SDK projects it seems to be set to x64/x86 etc, rather than AnyCPU. For the cross arm builds, we build the debuggees on windows and copy them to arm, so compiling not AnyCPU breaks that.